client/src/public/ts/socket.ts

```
@@ -115,6 +117,137 @@ socket.addEventListener("message", (event: any) => {
+          entryName.css("text-decoration", "underline");
+          entryName.css("cursor", "pointer");
+          variables.multiplayerLastGameRankings.playerListCache.registeredPlayers.add(
+            idName
+          );
+        }
+        entryLeft.append(entryName);
+        variables.multiplayerLastGameRankings.playerListCache.playerCount++;
+        //
+        const entryRight = $("<div></div>");
+        entryRight.addClass("ranking-placement--right");
+        const time = millisecondsToTime(parseInt(placement.time));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)